recipe/build_base.sh

```diff
@@ -41,11 +39,13 @@ else
 fi
 
 # Since these take very long to build in our emulated ci, disable for now
-if [[ ${target_platform} == linux-aarch64 ]]; then
-  _OPTIMIZED=no
-fi
-if [[ ${target_platform} == linux-ppc64le ]]; then
-  _OPTIMIZED=no
+if [[ ${CONDA_FORGE} == yes ]]; then
+  if [[ ${target_platform} == linux-aarch64 ]]; then
+    _OPTIMIZED=no
+  fi
+  if [[ ${target_platform} == linux-ppc64le ]]; then
+    _OPTIMIZED=no
+  fi
```

Comment:
Also seems like a good candidate for a `conda_build_config.yaml` value

Reply:
Same question.